README.md

```diff
@@ -2,6 +2,20 @@ Own Up Grown Up is a scheduling and reminder application that bridges the gap be
 
 Own Up Grown Up was created by TEAM JACK (Julianne Peters, Andrew Garrett, Chris Stubbs, and Kristen McCloud) and includes professional design and security.  Using our application and the Twilio API, you will always get your reminder and always be able to change your personal preferences.
 
-Don't forget to remember, or just use Own Up Grown Up. 
+Don't forget to remember, or just use Own Up Grown Up.
 # back2Point0
 # front2Point0
+
+
+TODO:
+  Same logic used in myTaskCard and taskCard to choose icons; we need to move up the state tree to DRY
+  Add id to <i> tags so that if no internet connection, still shows something
+  Move stylesheets to corresponding page folders
+  App.css has a .testing class - what is it used for?
```

Comment:
looks like this is just making the container for each task card and mytask card 80vw. we could add that property to the other classes those elements have and discard it. I bet it was named in a effort to problem solve ;-).